### PR TITLE
Added some examples

### PR DESCRIPTION
**Pull request by Tigerism**
Some beginners might find these to be helpful; hopefully, we'll have a voice example soon

### CODE REVIEW

File: examples/basicCommands.lua [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+discordia.extensions() -- load all helpful extensions
+
+
+client:on("ready", function() -- bot is ready
+	print("Logged in as " .. client.user.username)
+end)
+
+client:on("messageCreate", function(message)
+	local content = message.content
+	local author = message.author
```
Comment (SinisterRectus) [inline examples/basicCommands.lua]: `author` is unused.

File: examples/pingPong.lua [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+local client = discordia.Client()
+
+
+client:on("ready", function() -- bot is ready
+	print("Logged in as " .. client.user.username)
+end)
+
+client:on("messageCreate", function(message)
+	local content = message.content
+	local author = message.author
```
Comment (SinisterRectus) [inline examples/pingPong.lua]: `author` is unused.

File: examples/appender.lua [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+end)
+
+client:on("messageCreate", function(message)
+	local content = message.content
+	local author = message.author
+
+	if author == client.user then return end -- the bot should not append its own messages
+
+	if content == "!lines" then -- if the lines command is activated
+		message.channel:send {
```
Comment (SinisterRectus) [inline examples/appender.lua]: Should probably flush `lines` after it's sent.

**Comment by Tigerism** [comment]
Updated

**Pull request merged**